src/app/item-popup/ItemStat.m.scss

```diff
@@ -61,6 +62,85 @@
   background-color: white;
   color: black;
   transition: width 150ms ease-in-out;
+
+  &.negative {
+    background-color: #7a2727;
+    order: 1; // Push negative segments to the right side of the bar
+  }
+
+  // An assumption: never more than 4 part effects in a single stat?
+  // 3 have been observed on weapons with a barrel + mag + grip.
+  &.parts {
+    opacity: 0.88;
+    & + &.parts {
+      opacity: 0.76;
+      & + &.parts {
+        opacity: 0.64;
+        & + &.parts {
+          opacity: 0.52;
+        }
+      }
+    }
+  }
+
+  &.mod {
+    background-color: $stat-modded;
+    & + &.mod {
+      opacity: 0.8;
+      & + &.mod {
+        opacity: 0.6;
+      }
+    }
+  }
+
+  // Colors for the stat bars
+  &.masterwork {
+    background-color: $stat-masterworked;
+  }
+}
+
+.statBarTooltip {
+  display: grid;
+  grid-template-columns: auto auto;
+  column-gap: 4px;
+
+  & > :nth-child(2n-1) {
+    text-align: right;
+  }
+
+  .mod {
+    color: $stat-modded;
+  }
+  .masterwork {
+    color: $stat-masterworked;
+  }
+  .negative {
+    color: $red;
+    order: 1; // Push negative rows to the end of the addition list
+  }
+  .base {
+    opacity: 1;
+  }
+  .parts {
+    opacity: 0.8;
+  }
+
+  .tooltipNetStat {
```

Comment:
As a general rule, we don't need to nest selectors unless they have different properties when nested and when un-nested. Otherwise, since they're always uniquely-named, they won't ever conflict with anything else.

Reply:
Fair in tooltipNetStat, though some of these simpler class names like "negative" "mod" etc are doing double duty with nesting as bar colors and text colors.